puppet/modules/profiles/manifests/jenkins/packaging_node.pp

```diff
@@ -0,0 +1,19 @@
+# @summary A Jenkins packaging node
```

Comment:
Do we really need this and the unittest wrapper classes? Can't we set the booleans?

Reply:
we certainly can. I was mainly looking at this as "a profile should be something that almost never needs overrides", and overriding the booleans would mean the opposite ;)

Reply:
Looking at https://puppet.com/docs/pe/2019.2/the_roles_and_profiles_method.html I'd expect that more at the role level. Right now we haven't started with the role level (we barely have a profile layer).